src/components/Todo/Todo.js

```diff
@@ -71,16 +58,48 @@ const Todo = () => {
     setCount(count + 1);
   };
 
-    return (   
-      <div className={styles.wrap}>
-        <div className={styles.content}>
-          <h1 className={styles.title}>To Do List</h1>
-          <InputItem onClickAdd={onClickAdd} />
-          <ItemList items={items} onClickDone={onClickDone} onClickDelete = {onClickDelete} />
-          <Footer count = {count} />
+  const onClickFilter = filtration => setFilter(filtration);
+
+  let filter;
+    switch (filteredItems) {
+        case 'Завершенные':
+            filter = items.filter(item => item.isDone);
+            break;
+        case 'Незавершенные':
+            filter = items.filter(item => !item.isDone);
+            break;
+        default: 
+            filter = items;
+            break;
+   }
+   
+   const all = items;
+   const activeItems = items.filter(item => item.isDone === fasle);
```

Comment:
Codacy found an issue: ['fasle' is not defined.](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5830965)